l10n_co_res_partner/README.rst

```diff
@@ -0,0 +1,80 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+{module_title}
+==============
+
+This module extends the functionality of res.partner to support the requirements of the tax
+authority in Colombia
+
+Installation
```

Comment:
Fix the template part. 

Try to add a reationale as you did in the description of the PR about all what you added:

Recomendations.

1. Use links to Wikipedia or public colombia legislation to support any statement.
2. Avoid add features that are commonly accepted, just attach to features legally necessary.
3. If you want add features just because they are commonly accepted, may be this is not the repository, describe the feature and may be is better build a generic module in another repository.